tredparse/bam_parser.py

```diff
@@ -220,6 +220,9 @@ def parse(self, pad=SPAN):
                     if self.clip:
                         continue
                     try:
+                        if "nochr" in self.ref:
```

Comment:
@JiaShun-Xiao 
Probably a good idea to just precompute this in a class variable, for example:
```
self.ref = inputParams.ref
self.ref_nochr = "nochr" in self.ref
```
Then you can check:
```
if self.ref_nochr:
    ...
```